pydeconz/models/light/light.py

```diff
@@ -216,8 +237,12 @@ class LightEffect(enum.StrEnum):
     CANDLE = "candle"
     FIREPLACE = "fireplace"
     LOOP = "loop"
```

Comment:
Could you add a note on loop that it has been replaced with prism? Do you know from what hue fw its been replaced?

```suggestion
    LOOP = "loop"
    """Replaced by prism on later Hue firmwares."""
```

Reply:
I'm not exactly sure on the firmware version, it seems it depends on the kind of light and wasn't rolled out to all lights at the same time. I do have a reference to when deCONZ renamed the effect - I'll add a note referencing [that](https://github.com/dresden-elektronik/deconz-rest-plugin/pull/7206)